snmp/datadog_checks/snmp/data/profiles/_generic-if.yaml

```diff
@@ -66,6 +72,15 @@ metrics:
       name: ifAlias
     table: ifXTable
     tag: interface_alias
+  - column:
+      OID: 1.3.6.1.2.1.4.20.1.1
+      name: ipAdEntAddr
+      index_from_oid_value:
+        OID: 1.3.6.1.2.1.4.20.1.2
+        name: ipAdEntIfIndex
+    table: ipAddrTable
+    tag: interface_ip_address
```

Comment:
Note: for the same table ifTable, we have some metrics that require different `forced_type`. That's leading to duplication of `metric_tags` section. This can be solved by implementing `II-704`.